diff --git a/skia-bindings/Cargo.toml b/skia-bindings/Cargo.toml
index eca516a4..6a697344 100644
--- a/skia-bindings/Cargo.toml
+++ b/skia-bindings/Cargo.toml
@@ -36,7 +36,7 @@ skia = "m91-0.39.3"
 depot_tools = "73a2624"
 
 [features]
-default = ["binary-cache", "embed-icudtl"]
+default = ["binary-cache", "embed-icudtl", "gl", "vulkan", "d3d", "textlayout", "svg", "webp"]
 gl = []
 egl = []
 wayland = []
diff --git a/skia-org/Cargo.toml b/skia-org/Cargo.toml
index 3c085d6b..321d86b0 100644
--- a/skia-org/Cargo.toml
+++ b/skia-org/Cargo.toml
@@ -18,7 +18,7 @@ edition = "2018"
 default-run = "skia-org"
 
 [features]
-default = []
+default = ["gl", "vulkan", "d3d", "textlayout", "svg", "webp"]
 gl = ["offscreen_gl_context", "sparkle", "skia-safe/gl"]
 egl = ["gl", "skia-safe/egl"]
 x11 = ["gl", "skia-safe/x11"]
diff --git a/skia-safe/Cargo.toml b/skia-safe/Cargo.toml
index 6ddd6479..0f84729f 100644
--- a/skia-safe/Cargo.toml
+++ b/skia-safe/Cargo.toml
@@ -20,7 +20,7 @@ edition = "2018"
 doctest = false
 
 [features]
-default = ["binary-cache", "embed-icudtl"]
+default = ["binary-cache", "embed-icudtl", "gl", "vulkan", "d3d", "textlayout", "svg", "webp"]
 all-linux = ["gl", "egl", "vulkan", "x11", "wayland", "textlayout", "svg", "webp"]
 all-windows = ["gl", "vulkan", "d3d", "textlayout", "svg", "webp"]
 all-macos = ["gl", "vulkan", "metal", "textlayout", "svg", "webp"]
