diff --git a/BUILD.gn b/BUILD.gn
index b5929c6744..efdd6bd168 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -1087,9 +1087,9 @@ group("modules") {
     "modules/skottie",
     "modules/skshaper",
   ]
-  if (target_cpu == "wasm") {
+  # Patched in by skia-bindings
+  # **SKIA-BINDINGS-PATCH-MARKER-SKPARAGRAPH**
     deps += [ "modules/skparagraph" ]
-  }
 }
 
 # Targets guarded by skia_enable_tools may use //third_party freely.
diff --git a/modules/skparagraph/BUILD.gn b/modules/skparagraph/BUILD.gn
index 5a85d37c4e..a3ab9b8846 100644
--- a/modules/skparagraph/BUILD.gn
+++ b/modules/skparagraph/BUILD.gn
@@ -18,6 +18,7 @@ if (skia_enable_skparagraph) {
 
   component("skparagraph") {
     import("skparagraph.gni")
+    complete_static_lib = false
     public_configs = [ ":public_config" ]
     public = skparagraph_public
     if (skia_use_icu && skia_use_harfbuzz) {
