diff --git a/BUILD.gn b/BUILD.gn
index 6a85dd96b..809d4724d 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -1076,6 +1076,9 @@ group("modules") {
     "modules/particles",
     "modules/skottie",
     "modules/skshaper",
+    # Patched in by skia-bindings
+    # **SKIA-BINDINGS-PATCH-MARKER-SKPARAGRAPH**
+    "modules/skparagraph",
   ]
 }
 
diff --git a/modules/skparagraph/BUILD.gn b/modules/skparagraph/BUILD.gn
index 5a85d37c4..a3ab9b884 100644
--- a/modules/skparagraph/BUILD.gn
+++ b/modules/skparagraph/BUILD.gn
@@ -18,6 +18,7 @@ if (skia_enable_skparagraph) {
 
   component("skparagraph") {
     import("skparagraph.gni")
+    complete_static_lib = false
     public_configs = [ ":public_config" ]
     public = skparagraph_public
     if (skia_use_icu && skia_use_harfbuzz) {
